.github/workflows/mariadb.yml

```diff
@@ -0,0 +1,95 @@
+name: MariaDB
+on:
+  push:
+    paths:
+      - "**/*.cpp"
+      - "**/*.h"
+      - "**/*.hpp"
+      - "**/CMakeLists.txt"
+      - "**/Makefile.am"
+      - ".github/workflows/mariadb.yml"
+      - "configure.ac"
+      - "doc/**"
+      - "mysql-test/**"
+      - "packages/**"
+      - "version_*"
+  pull_request:
+    paths:
+      - "**/*.cpp"
+      - "**/*.h"
+      - "**/*.hpp"
+      - "**/CMakeLists.txt"
+      - "**/Makefile.am"
+      - ".github/workflows/mariadb.yml"
+      - "configure.ac"
+      - "doc/**"
+      - "mysql-test/**"
+      - "packages/**"
+      - "version_*"
+  schedule:
+    - cron: |
+        0 0 * * *
```

Comment:
Rather than a daily schedule (note updates rarely occur daily), we could in MariaDB's CI trigger this test.

We currently do this with MariaDB-operator  like https://buildbot.mariadb.org/#/builders/311/builds/28724 "test operator with image" which invokes https://github.com/mariadb-operator/mariadb-operator/blob/main/.github/workflows/test-image-ubi.yml.

Note the "Tell the MariaDB Folks that failed" on failure step that can, with a provided token, give us a callback if we created an incompatibility.

Ping me on Zulip if this is something you are interested in and we can exchange some tokens.

Reply:
Thanks for the suggestion. I'll reach out again once we have the capability to run tests on MariaDB main so we can discuss integrating this further and coordinate the token exchange.